portfolio/src/main/webapp/script.js

```diff
@@ -11,3 +11,43 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+/** Fetches existing comments and updates UI. */
+function getComments() {
+    document.getElementById('comments').innerHTML = "";
+    var maxComments = document.getElementById("maxComments").value;
+    fetch('/data?max=' + maxComments).then(response => response.json()).then(comments => {
```

Comment:
You can try use JavaScript template syntax here.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Thanks!